gcloud-java-core/src/main/java/com/google/gcloud/RetryParams.java

```diff
@@ -48,12 +48,16 @@
 
   private static final long serialVersionUID = -8492751576749007700L;
 
+  /**
+   * Note that App Engine Standard Environment front-end modules have a 60 second deadline for HTTP
+   * requests. For that reason, we set the default total retry period to less than 60 seconds.
+   */
+  public static final long DEFAULT_TOTAL_RETRY_PERIOD_MILLIS = 50_000L;
```

Comment:
I am fine with that though I thought your preference was to make it 80_000 *and* add to the APIs javadoc the suggestion to lower it if planned to be used on an AE front-end.